testsuite/domain/src/test/resources/domain-configs/domain-standard.xml

```diff
@@ -300,8 +300,8 @@
             <subsystem xmlns="urn:jboss:domain:messaging-activemq:13.0">
                 <server name="default"
                         persistence-enabled="false">
-                    <journal file-size="1024" />
                     <security elytron-domain="ApplicationDomain"/>
+                    <journal file-size="1024" />
```

Comment:
Could you apply the same fix L660 and also at domain-standard-ee.xml:589?

Reply:
Done